contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -459,7 +454,7 @@ def getDetectionLookups(cls, v: list[str], info: ValidationInfo) -> list[Lookup]
 
         director: DirectorOutputDto = info.context.get("output_dto", None)
 
-        search: Union[str, dict] = info.data.get("search", None)                                    # type: ignore
+        search: Union[str, dict[str,Any],None] = info.data.get("search", None)                                    # type: ignore
```

Comment:
nit: can we make this change PEP compliant (spaces after `,`)?

```
search: Union[str, dict[str, Any], None] = info.data.get("search", None)
```

Reply:
Yup, agreed and changed :)